api/v1alpha3/machine_types.go

```diff
@@ -155,6 +155,10 @@ type MachineStatus struct {
 	// InfrastructureReady is the state of the infrastructure provider.
 	// +optional
 	InfrastructureReady bool `json:"infrastructureReady"`
+
+	// ObservedGeneration is the latest generation observed by the controller.
+	// +optional
```

Comment:
Should this be no longer optional, since we are not setting omitempty? https://github.com/eBay/Kubernetes/blob/master/docs/devel/api-conventions.md#optional-vs-required